### PR TITLE
Adds WebGL2 announcement to CHANGELOG

### PR DESCRIPTION
**Pull request by sanjeetsuhag**
See https://github.com/CesiumGS/cesium/pull/10894

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @sanjeetsuhag!
* :heavy_check_mark: Signed CLA found.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

**Review by ggetz — COMMENTED** [review]
Thanks @sanjeetsuhag!

**Comment by sanjeetsuhag** [comment]
@ggetz Updated. Since we're replacing `ContextOptions.requestWebgl2` with  `ContextOptions.requestWebgl1` should we add a separate deprecation notice for that?

**Comment by ggetz** [comment]
Yes, let's do that! It is [an official part pf the API](https://cesium.com/learn/cesiumjs/ref-doc/global.html#ContextOptions).

**Comment by sanjeetsuhag** [comment]
@ggetz Updated the PR.
- Added `ContextOptions.requestWebgl1`
- Deprecated `ContextOptions.requestWebgl2`
- Updates CHANGES.md

File: packages/engine/Source/Renderer/Context.js [hunk 1]
```diff
@@ -42,9 +43,18 @@ function Context(canvas, options) {
   Check.defined("canvas", canvas);
   //>>includeEnd('debug');
 
+  let requestWebgl2 = false;
+  if (defined(options) && defined(options.requestWebgl2)) {
+    requestWebgl2 = options.requestWebgl2;
+    Context._deprecationWarning(
+      "ContextOptions.requestWebgl2",
+      "Cesium.requestWebgl2 was deprecated in CesiumJS 1.101 and will be removed in 1.102. Use Cesium.requestWebgl1 to request a WebGL1 or WebGL2 context."
```
Comment (ggetz) [inline packages/engine/Source/Renderer/Context.js:51]:
```suggestion
      "Cesium.requestWebgl2 was deprecated in CesiumJS 1.101 and will be removed in 1.102. Instead, CesiumJS will default to using a WebGL2 context for rendering. Use Cesium.requestWebgl1 to request a WebGL1 or WebGL2 context."
```

How about this, just to provide a bit more guidance if a user sees this message.

File: packages/engine/Specs/Renderer/ContextSpec.js [hunk 2]
```diff
@@ -325,6 +327,36 @@ describe(
...
+
+      if (!webglStub) {
+        const c2 = createContext({
+          requestWebgl1: false,
+        });
+        expect(c2._webgl2).toBe(true);
+      }
+    });
+
+    it("prints deprecation warning when using requestWebgl2 instead of requestWebgl1", function () {
```
Comment (ggetz) [inline packages/engine/Specs/Renderer/ContextSpec.js:345]: Great that this was tested explicitly, but typically there are no explicit tests for deprecation warnings as they will need to be removed.

File: packages/engine/Source/Renderer/Context.js (hunk 1 above)
Comment (sanjeetsuhag) [reply packages/engine/Source/Renderer/Context.js:51]: That's better, updated.

File: packages/engine/Specs/Renderer/ContextSpec.js (hunk 2 above)
Comment (sanjeetsuhag) [reply packages/engine/Specs/Renderer/ContextSpec.js:345]: Do you recommend removing this spec or should I create an issue to `remove in 1.102` and address it the WebGL2 by default PR? I also wanted to add a spec to ensure the now deprecated `requestWebgl2` was selecting the context correctly.

Comment (ggetz) [reply packages/engine/Specs/Renderer/ContextSpec.js:345]: I think remove it. And the WebGL2 by default PR should be testing this new option instead.

Comment (sanjeetsuhag) [reply packages/engine/Specs/Renderer/ContextSpec.js]: Updated.

**Comment by ggetz** [comment]
Great, thanks @sanjeetsuhag!

**Pull request merged**